test/Microsoft.ML.TestFramework/ModelHelper.cs

```diff
@@ -40,24 +41,187 @@ public static void WriteKcHousePriceModel(string dataPath, Stream stream)
 
         public static IDataView GetKcHouseDataView(string dataPath)
         {
-            var dataSchema = "col=Id:TX:0 col=Date:TX:1 col=Label:R4:2 col=Bedrooms:R4:3 col=Bathrooms:R4:4 col=SqftLiving:R4:5 col=SqftLot:R4:6 col=Floors:R4:7 col=Waterfront:R4:8 col=View:R4:9 col=Condition:R4:10 col=Grade:R4:11 col=SqftAbove:R4:12 col=SqftBasement:R4:13 col=YearBuilt:R4:14 col=YearRenovated:R4:15 col=Zipcode:R4:16 col=Lat:R4:17 col=Long:R4:18 col=SqftLiving15:R4:19 col=SqftLot15:R4:20 header+ sep=,";
-            var txtArgs = new TextLoader.Arguments();
+            var dataSchema = "col=Id:TX:0 col=Date:TX:1 col=Label:R4:2 col=Bedrooms:R4:3 " +
+                "col=Bathrooms:R4:4 col=SqftLiving:R4:5 col=SqftLot:R4:6 col=Floors:R4:7 " +
+                "col=Waterfront:R4:8 col=View:R4:9 col=Condition:R4:10 col=Grade:R4:11 " +
+                "col=SqftAbove:R4:12 col=SqftBasement:R4:13 col=YearBuilt:R4:14 " +
+                "col=YearRenovated:R4:15 col=Zipcode:R4:16 col=Lat:R4:17 col=Long:R4:18 " +
+                "col=SqftLiving15:R4:19 col=SqftLot15:R4:20 header+ sep=,";
+
+            var txtArgs = new Runtime.Data.TextLoader.Arguments();
             bool parsed = CmdParser.ParseArguments(s_environment, dataSchema, txtArgs);
             s_environment.Assert(parsed);
-            var txtLoader = new TextLoader(s_environment, txtArgs, new MultiFileSource(dataPath));
+            var txtLoader = new Runtime.Data.TextLoader(s_environment, txtArgs, new MultiFileSource(dataPath));
             return txtLoader;
         }
 
         private static ITransformModel CreateKcHousePricePredictorModel(string dataPath)
         {
-            var dataSchema = "col=Id:TX:0 col=Date:TX:1 col=Label:R4:2 col=Bedrooms:R4:3 col=Bathrooms:R4:4 col=SqftLiving:R4:5 col=SqftLot:R4:6 col=Floors:R4:7 col=Waterfront:R4:8 col=View:R4:9 col=Condition:R4:10 col=Grade:R4:11 col=SqftAbove:R4:12 col=SqftBasement:R4:13 col=YearBuilt:R4:14 col=YearRenovated:R4:15 col=Zipcode:R4:16 col=Lat:R4:17 col=Long:R4:18 col=SqftLiving15:R4:19 col=SqftLot15:R4:20 header+ sep=,";
-
             Experiment experiment = s_environment.CreateExperiment();
 
-            var importData = new Data.TextLoader();
-            importData.CustomSchema = dataSchema;
-            Data.TextLoader.Output imported = experiment.Add(importData);
+            var importData = new Data.TextLoader(dataPath)
+            {
+                Arguments = new TextLoaderArguments
+                {
+                    Delimiter = ',',
+                    HasHeader = true,
+                    Column = new[]
+                    {
+                        new TextLoaderColumn()
+                        {
+                            Name = "Id",
+                            Source = new [] { new TextLoaderRange() { Min = 0, Max = 0} },
```

Comment:
>  [](start = 40, length = 1)

Maybe VS could format the file to clean up thse whitespace issues, I think default VS formatting is `new[]` not `new []`.

Reply:
I tried ctrl-k-d but still there is space.

---
In reply to: [188013046](https://github.com/dotnet/machinelearning/pull/142#discussion_r188013046) [](ancestors = 188013046)

Reply:
Ah. How strange. Oh well.

---
In reply to: [188121093](https://github.com/dotnet/machinelearning/pull/142#discussion_r188121093) [](ancestors = 188121093,188013046)